.github/vale-styles/aws-products/company-name.yml

```diff
@@ -0,0 +1,276 @@
+# This style rule ensures that mentions of AWS product names are correct.
+# To get the list of service names:
+#
+# - Open https://health.aws.amazon.com/health/status in a browser.
+# - Look at the "Network" pane in dev tools and find the URL
+#   that returned services.json.
+# - Save services.json to a local file. 
+# - Run the following command to get a list of product names that begin with
+#   "AWS" or "Amazon", formatted as a mapping where each key is the incorrect
+#   usage and each value is the correct usage. Each mapping is indented by two
+#   spaces so we can use it as the value of "swap":
+#
+#   jq -r '.[].service_name' < services.json | sort | uniq | grep -E "^(AWS|Amazon)" | gawk '
```

Comment:
Just suggestions. No worries if you prefer the way it is.

Reply:
`sort -u` is also a drop-in replacement for `sort | uniq`, since we entered into this territory. I didn't review the comment script, but I trust Cam to be correct.  :)

Reply:
Thanks! Changed